JAVA/Ping pong/README.md

```diff
@@ -0,0 +1,33 @@
+# A Ping Pong Java Game
+
+- It is an endless 2 player Ping Pong game made using Java Swings that is played on the same device.
+
+Let's try out yourself😉
+<br>
+But 1st let's have a quick start:
+
+## Quick Start
+### Clone this repository
+``` 
+git clone https://github.com/abhijeet007rocks8/Dev-Scripts.git
+```
+
+### Change Directory
+```
+cd Java
+```
+```
+cd Ping pong
+
+```
+
+### Run the application
+
+1. Open the "Ping Pong" folder as a project in your IDE.
+2. Run the PongGame.java program.
+3. Enjoy playing using W and S keys for Player 1. Up and Down keys for Player 2.✌
+4. Also you can see the time recorded for the packet of information makes during its route to the destination.
+
+### Hope you have a lot of fun.
+
+# Happy Playing✨🎉
```

Comment:
for this follow all the rules which are mentioned in the #43 .
- mention tech Stack used 
- what are the dependencies are needed to be installed
- mention basic components of the project
- add demonstration video